vpr/src/device/rr_graph_builder.h

```diff
@@ -40,6 +38,63 @@ class RRGraphBuilder {
     t_rr_graph_storage& node_storage();
     /** @brief Return a writable object for update the fast look-up of rr_node */
     RRSpatialLookup& node_lookup();
+    /** @brief Return a writable object for rr_node_metadata */
+    MetadataStorage<int>& rr_node_metadata();
+    /** @brief Return a writable object for rr_edge_metadata */
+    MetadataStorage<std::tuple<int, int, short>>& rr_edge_metadata();
+
+    MetadataStorage<int> rr_node_metadata_data() const;
+
+    MetadataStorage<std::tuple<int, int, short>> rr_edge_metadata_data() const;
+    //rr_node_metadata_clear
+    inline void rr_node_metadata_clear() {
```

Comment:
Our naming convention is ``clear_rr_node_metadata()``

See example: 

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/36789e456ab292038b8518bf2c835c9b9dcaaa0e/vpr/src/device/rr_graph_obj.h#L718-L722

Reply:
Changed as per naming convention.